README.md

```diff
@@ -44,17 +44,17 @@ Azure Container Service (ACS) and Kubernetes as the Container Orchestrator.
 How to use it
 =============
 
-To run Remote Monitoring on your local machine:
+The easiest way to get started with Remote Monitoring is to deploy the running application to your Azure subscription through the command line interface (CLI). Deployment instructions for the CLI are in the CLI repo: https://github.com/Azure/pcs-cli.  These instructions will show you how to deploy all Remote Monitoring resources, with seed data, and see a running solution.  You can then customize, extend, or replace the components of the system to meet your needs.  The Developer Reference Guide here: https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide will facilitate customization by showing how to replace the containers deployed by the CLI with your own.
```

Comment:
could you split lines? it's easier to add PR comments on the right point

Reply:
I'd add a link to the first occurrence of "command line interface", pointing to the CLI installation docs

Reply:
nit: "repo" => "repository"

Reply:
I'd prefer to link text rather than showing the full link in the text, I find it easier to follow/browse